### PR TITLE
Update dependencies to django 2.2.13

### PR DESCRIPTION
**Pull request by sposs**
Bump other packages + fix permission conflicts

### CODE REVIEW

**Comment by canassa** [comment]
That's great! I wonder while the GitHub actions aren't running on your PR. Let me check that.

**Comment by canassa** [comment]
Alright, the pipeline should be working now

**Comment by sposs** [comment]
I saw it fail, but I don't get why: seems a low level error when attempting to write the test DB. I have a hard time with that: on my local machine all the tests pass.

**Comment by sposs** [comment]
py run-test: commands[1] | py.test tests --cov localshop --cov-report term-missing
============================================================================================ test session starts =============================================================================================
platform linux -- Python 3.6.9, pytest-5.4.3, py-1.8.1, pluggy-0.13.1
cachedir: .tox/py/.pytest_cache
django: settings: localshop.settings (from env)
rootdir: /home/stephane/PycharmProjects/localshop, inifile: setup.cfg
plugins: django-webtest-1.9.7, Faker-4.1.0, celery-4.4.0, cov-2.9.0, requests-mock-1.8.0, django-3.9.0
collected 73 items                                                                                                                                                                                           

tests/apps/accounts/test_forms.py .....                                                                                                                                                                [  6%]
tests/apps/dashboard/test_forms.py ......                                                                                                                                                              [ 15%]
tests/apps/packages/test_fetch_package_task.py .....                                                                                                                                                   [ 21%]
tests/apps/packages/test_models.py ......s                                                                                                                                                             [ 31%]
tests/apps/packages/test_tasks.py .....                                                                                                                                                                [ 38%]
tests/apps/packages/test_xmlrpc.py ..........                                                                                                                                                          [ 52%]
tests/apps/packages/views/test_download_file.py .....                                                                                                                                                  [ 58%]
tests/apps/packages/views/test_simple_detail.py ....                                                                                                                                                   [ 64%]
tests/apps/packages/views/test_simple_index.py ..............                                                                                                                                          [ 83%]
tests/apps/permissions/test_models.py ...                                                                                                                                                              [ 87%]
tests/apps/permissions/test_utils.py ....                                                                                                                                                              [ 93%]
tests/management/test_command_init.py .                                                                                                                                                                [ 94%]
tests/apps/packages/test_pypi.py .                                                                                                                                                                     [ 95%]
tests/apps/packages/test_utils.py ..                                                                                                                                                                   [ 98%]
tests/management/test_runner.py .                                                                                                                                                                      [100%]

----------- coverage: platform linux, python 3.6.9-final-0 -----------
Name                                                       Stmts   Miss  Cover   Missing
----------------------------------------------------------------------------------------
src/localshop/__init__.py                                      1      0   100%
src/localshop/apps/__init__.py                                 0      0   100%
src/localshop/apps/accounts/__init__.py                        0      0   100%
src/localshop/apps/accounts/admin.py                          55     11    80%   40-44, 48-52, 76
src/localshop/apps/accounts/auth_urls.py                      10      1    90%   22
src/localshop/apps/accounts/forms.py                          47      0   100%
src/localshop/apps/accounts/models.py                         45      5    89%   20, 65, 68, 71, 90
src/localshop/apps/accounts/urls.py                            4      0   100%
src/localshop/apps/accounts/views.py                         142     60    58%   31-33, 36, 44-48, 57, 65, 70-71, 74-76, 84-85, 88, 96-97, 100, 108-109, 112-114, 117, 124, 133-134, 137, 140, 146, 149-152, 161-162, 165, 168, 176, 179, 194-229
src/localshop/apps/dashboard/__init__.py                       0      0   100%
src/localshop/apps/dashboard/forms.py                         49      0   100%
src/localshop/apps/dashboard/urls.py                           5      0   100%
src/localshop/apps/dashboard/views/__init__.py                 5      0   100%
src/localshop/apps/dashboard/views/cidr.py                    31      7    77%   20, 28, 33, 41, 46, 54, 59
src/localshop/apps/dashboard/views/credentials.py             45     13    71%   25, 28-30, 38, 41, 49-53, 63, 66, 76
src/localshop/apps/dashboard/views/misc.py                    28     12    57%   16, 22-31, 39, 44-45
src/localshop/apps/dashboard/views/package.py                 50     20    60%   26-32, 35-36, 39, 49-57, 65-67, 78, 81-83
src/localshop/apps/dashboard/views/repository.py              74     26    65%   26, 41-50, 53-55, 59-60, 71-77, 88, 91, 94-96, 105, 109, 119-122
src/localshop/apps/packages/__init__.py                        0      0   100%
src/localshop/apps/packages/admin.py                          26      0   100%
src/localshop/apps/packages/forms.py                          49      2    96%   51, 79
src/localshop/apps/packages/mixins.py                          6      0   100%
src/localshop/apps/packages/models.py                        137     19    86%   46, 54-56, 63, 73, 81, 113, 116, 129, 166, 170-176, 229, 263
src/localshop/apps/packages/pypi.py                           20      0   100%
src/localshop/apps/packages/tasks.py                          83      0   100%
src/localshop/apps/packages/urls.py                            5      0   100%
src/localshop/apps/packages/utils.py                          65     10    85%   42, 66, 69, 78, 82-83, 105-106, 110-111
src/localshop/apps/packages/views.py                         139     11    92%   58, 63, 70, 91, 130, 141, 202, 223-225, 233
src/localshop/apps/packages/xmlrpc.py                         36      1    97%   11
src/localshop/apps/permissions/__init__.py                     0      0   100%
src/localshop/apps/permissions/admin.py                        9      0   100%
src/localshop/apps/permissions/mixins.py                      62     12    81%   43-47, 59-62, 92-93, 100-107
src/localshop/apps/permissions/models.py                      37      2    95%   31, 75
src/localshop/apps/permissions/utils.py                       54     16    70%   36-45, 74-82
src/localshop/celery.py                                        6      0   100%
src/localshop/http.py                                         10      0   100%
src/localshop/management/__init__.py                           0      0   100%
src/localshop/management/commands/__init__.py                  0      0   100%
src/localshop/management/commands/create_default_user.py      14     14     0%   1-24
src/localshop/management/commands/init.py                      6      0   100%
src/localshop/management/commands/repository_refresh.py        5      5     0%   1-9
src/localshop/runner.py                                        6      0   100%
src/localshop/settings/__init__.py                             1      0   100%
src/localshop/settings/defaults.py                            73      8    89%   192-196, 208-211
src/localshop/templatetags/__init__.py                         0      0   100%
src/localshop/templatetags/forms.py                            4      1    75%   8
src/localshop/templatetags/permission_tags.py                  4      1    75%   8
src/localshop/urls.py                                         14      0   100%
src/localshop/utils.py                                        22      2    91%   18-19
src/localshop/views.py                                         7      3    57%   9-11
src/localshop/wsgi.py                                          4      4     0%   1-7
----------------------------------------------------------------------------------------
TOTAL                                                       1495    266    82%


======================================================================================= 72 passed, 1 skipped in 6.51s ========================================================================================
__________________________________________________________________________________________________ summary ___________________________________________________________________________________________________
  py: commands succeeded
  congratulations :)

**Comment by sposs** [comment]
I had a migration file not added/commited...

File: src/localshop/apps/packages/models.py [hunk 1]
```diff
@@ -91,7 +90,7 @@ class Package(models.Model):
 
     name = models.CharField(max_length=200, db_index=True, validators=[
         RegexValidator(
-            re.compile(r'^[-a-zA-Z0-9_\.]+\Z'),
+            re.compile(r'^[-a-zA-Z0-9_.]+\Z'),
```
[the remark this answers is not in the document]
Comment (sposs) [reply src/localshop/apps/packages/models.py:93]: In a character set (in []), it is not necessary to escape anything. https://stackoverflow.com/questions/10593387/when-do-i-need-to-escape-characters-within-a-regex-character-set-within

[the remark this answers is not in the document]
Comment (canassa) [reply src/localshop/apps/packages/models.py:93]: Cool! thanks

**Pull request merged**